docs/getting-started.md

```diff
@@ -305,3 +305,20 @@ coap get coap://localhost/channels/<channel_id>/messages?key=<client_token> -o
 
 where `-o` in indicates that value of `Observe` option is 0.
 To unsubscribe from channel, client should send same message with value of `Observe` option set to 1, ignore confirmable message received from service (by default, server will attempt to send confirmable message multiple times, as described in [RFC 7252](https://tools.ietf.org/html/rfc7252#page-28) or send `Reset` message with appropriate token.
+
+### MQTT
+
+To send and receive messages over MQTT you could use [Mosquitto](https://mosquitto.org),
```

Comment:
Mosquitto tools, not just "Mosquitto" - because Mosquitto is MQTT broker, and Mosquitto tools are CLI tools `mosquitto_pub` and `mosquitto_sub`.

You can keep the same link, just add "tools"